store/backup.go

```diff
@@ -172,6 +172,23 @@ func (s *Store) PatchBackup(ctx context.Context, patch *api.BackupPatch) (*api.B
 	return backup, nil
 }
 
+// FindBackupSettingByInstanceID finds a list of BackupSetting of databases in the instance.
+func (s *Store) FindBackupSettingByInstanceID(ctx context.Context, instanceID int) ([]*api.BackupSetting, error) {
```

Comment:
Can we use the semantics of FindBackupSetting(context, find) so that we can scale to other find use cases later?

Reply:
SG.